media/css/pocket/components/new-tab-learn-more.scss

```diff
@@ -70,12 +70,13 @@ $image-path: '/media/protocol/img';
     }
 }
 
-.section-top-header-lede {
+.tagline {
```

Comment:
Updated classname and style to match https://github.com/mozilla/bedrock/blob/main/bedrock/pocket/templates/pocket/pocket-and-firefox.html#L28